### PR TITLE
jsonnet: bump telemeter

### PR DESCRIPTION
**Pull request by squat**
blocked: https://github.com/openshift/telemeter/pull/148

Bump Telemeter client to collect the newly added infrastructure metrics in #291.

@s-urbaniak @metalmatze

### CODE REVIEW

File: assets/telemeter-client/secret.yaml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 apiVersion: v1
 data:
-  match-rules: e19fbmFtZV9fPSJ1cCJ9CntfX25hbWVfXz0iY2x1c3Rlcl92ZXJzaW9uIn0Ke19fbmFtZV9fPSJjbHVzdGVyX3ZlcnNpb25fYXZhaWxhYmxlX3VwZGF0ZXMifQp7X19uYW1lX189ImNsdXN0ZXJfb3BlcmF0b3JfdXAifQp7X19uYW1lX189ImNsdXN0ZXJfb3BlcmF0b3JfY29uZGl0aW9ucyJ9CntfX25hbWVfXz0iY2x1c3Rlcl92ZXJzaW9uX3BheWxvYWQifQp7X19uYW1lX189ImNsdXN0ZXJfdmVyc2lvbl9wYXlsb2FkX2Vycm9ycyJ9CntfX25hbWVfXz0ibWFjaGluZV9jcHVfY29yZXMifQp7X19uYW1lX189Im1hY2hpbmVfbWVtb3J5X2J5dGVzIn0Ke19fbmFtZV9fPSJldGNkX29iamVjdF9jb3VudHMifQp7X19uYW1lX189IkFMRVJUUyIsYWxlcnRzdGF0ZT0iZmlyaW5nIn0Ke19fbmFtZV9fPSJjb2RlOmFwaXNlcnZlcl9yZXF1ZXN0X2NvdW50OnJhdGU6c3VtIn0Ke19fbmFtZV9fPSJrdWJlX3BvZF9zdGF0dXNfcmVhZHk6ZXRjZDpzdW0ifQp7X19uYW1lX189Imt1YmVfcG9kX3N0YXR1c19yZWFkeTppbWFnZV9yZWdpc3RyeTpzdW0ifQ==
```
Comment (metalmatze) [inline assets/telemeter-client/secret.yaml]: You still need to this file to populated with a salt and token, right?

Comment (squat) [reply assets/telemeter-client/secret.yaml]: yes, the salt and token are still secret and need to be kept here.

**Comment by squat** [comment]
@s-urbaniak no need to restart tests until the telemeter Pr merged. Telemeter client is essentially broken until then and the tests *should* fail

**Pull request merged**